packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -164,6 +164,11 @@ function Session(props: PropsWithChildren<State>) {
         isSharingDirectory={isSharingDirectory}
         onShareDirectory={() => {
           setIsSharingDirectory(true);
+          const sharedDirHandle = window
+            .showDirectoryPicker()
```

Comment:
Do we need to handle browsers which don't support this API?

Reply:
Yeah, I left that as a "known issue" for now: https://github.com/gravitational/webapps/issues/959